sdk/monitor/opentelemetry-exporter-azure-monitor/src/export/trace.ts

```diff
@@ -134,9 +140,20 @@ export class AzureMonitorTraceExporter implements SpanExporter {
     resultCallback(await this.exportEnvelopes(envelopes));
   }
 
-  shutdown(): void {
+  shutdown(): Promise<void> {
     this._logger.info("Azure Monitor Trace Exporter shutting down");
-    this._sender.shutdown();
+    return new Promise((resolve, reject) => {
+      Promise.resolve()
+        .then(() => {
+          return this._sender.shutdown();
+        })
+        .then(() => {
+          resolve();
+        })
+        .catch((e) => {
+          reject(e);
+        });
+    });
```

Comment:
The above code doesn't make sense to me. If we actually used the result I'd say to make it an async function and await it, but since we don't, we can just return the sender shutdown method and update that interface to return a Promise, yeah?

```suggestion
    return this._sender.shutdown();
```